rcgen/src/key_pair.rs

```diff
@@ -211,6 +229,12 @@ impl KeyPair {
 		}
 	}
 
+	/// Generate a new random key pair for the specified signature algorithm
+	#[cfg(not(feature = "crypto"))]
+	pub fn generate(_alg: &'static SignatureAlgorithm) -> Result<Self, Error> {
```

Comment:
Original comment by @djc: https://github.com/rustls/rcgen/pull/207#discussion_r1450126844

> It doesn't seem like a great idea to generate an impl that can only error?

Would you rather I remove this entirely and the `generate` just isn't available without the `crypto` feature?

Reply:
yes, that's generally preferable. Rust has strong static checks. the std implementations for wasm are panicking, and now I heard that it is considered to be a mistake.

Reply:
I see. Well, this has already been done.

Reply:
Did a commit for this change not get pushed or am I confused about the outcome of the discussion? 

The state of the branch at the moment is that there is still a `generate` impl gated by `#[cfg(not(feature = "crypto"))]` that will only ever produce an error. I read the above discussion as preferring this impl not exist.

Reply:
@corrideat I think this piece of feedback is the only merge blocker. Do you think you'll have a chance to fix it?